ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -280,27 +294,10 @@ export const NetworkListMenu = ({ onClose }) => {
           {t('networkMenuHeading')}
         </ModalHeader>
         <Box className="multichain-network-list-menu">
-          {showSearch ? (
```

Comment:
Does this need to be moved to a different component?  This likely created the scrolling?  Maybe restore the `<Box>`?